### DISCUSSION TITLE
style

### DISCUSSION THREAD

**Discussion by L-Qun** [jotaijs/jotai-devtools#117] [Q&A]
I don't know why the .jotai-devtools-trigger-button style is being overridden by .jotai-devtools-1xwu97l, causing it not to be in the correct position.
![image](https://github.com/jotaijs/jotai-devtools/assets/68707557/a33e693a-d326-486b-8c1c-44c74433fc03)

**Comment by arjunvegda** [comment]
Hm interesting! Looks like a CSS specificity issue. Is this reproducible?

**Reply by L-Qun** [reply]
It can only be reproduced in my project

**Reply by L-Qun** [reply]
Are there any possible ways to solve this problem?

**Reply by arjunvegda** [reply]
Are you using an emotion cache in your project? It would be awesome if you could create a small repro on [csb](https://csb.jotai.org/). I can help take a look
